x-pack/plugin/repository-encrypted/src/main/java/org/elasticsearch/repositories/encrypted/DecryptionPacketsInputStream.java

```diff
@@ -123,6 +123,12 @@ public void reset() throws IOException {
         throw new IOException("Mark/reset not supported");
     }
 
+    @Override
+    public void close() throws IOException {
+        super.close();
```

Comment:
You might want `org.elasticsearch.core.internal.io.IOUtils#close(java.io.Closeable...)` here as well to safely close `super` and `source`?

Reply:
Yeah, I've pushed 8e036f0 !